dbt_copilot_helper/addon-plans.yml

```diff
@@ -1,187 +1,186 @@
+  # 6.38GB RAM, highly-available
   medium-ha:
     replicas: 2
     instance: 'cache.m6g.large'
 
   # 12.93GB RAM, single node, no failover.
   large:
     replicas: 0
-    instance: 'cache.m6g.xlarge'
+    instance: cache.m6g.xlarge
 
   # 12.93GB RAM, highly-available
   large-ha:
     replicas: 2
-    instance: 'cache.m6g.xlarge'
+    instance: cache.m6g.xlarge
 
   # 26.04GB RAM, single node, no failover.
   x-large:
     replicas: 0
-    instance: 'cache.m6g.2xlarge'
+    instance: cache.m6g.2xlarge
 
   # 26.04GB RAM, highly-available
   x-large-ha:
     replicas: 2
-    instance: 'cache.m6g.2xlarge'
+    instance: cache.m6g.2xlarge
 
 opensearch:
-  # 1 dedicated VM, 1 CPU per VM, 4GB RAM per VM, 80GB disk space.
+  # 2 vCPU, 2GB RAM, volume size range 10-100GB
   tiny:
     volume-size: 80
     instances: 1
     master: false
     instance: t3.small.search
 
-  # 1 CPU per VM, 4GB RAM per VM, 80GB disk space
+  # 2 vCPU, 4GB RAM, volume size range 10-200GB
   small:
     volume-size: 200
     instances: 1
     master: false
     instance: t3.medium.search
 
-  # 3 dedicated VMs, 1 CPU per VM, 4GB RAM per VM, 80GB disk space
+  # 2 nodes with 2 vCPU, 4GB RAM, volume size range 10-200GB
   small-ha:
     volume-size: 200
     instances: 2
     master: false
     instance: t3.medium.search
 
-  # 3 dedicated VMs, 2 CPU per VM, 8GB RAM per VM, 525GB disk space
+  # 2 vCPU, 8GB RAM, volume size range 10-512GB
   medium:
-    volume-size: 525
+    volume-size: 512
     instances: 1
     master: false
     instance: m6g.large.search
 
-  # 3 dedicated VMs, 2 CPU per VM, 8GB RAM per VM, 525GB disk space
+  # 2 nodes with 2 vCPU, 8GB RAM, volume size range 10-512GB
   medium-ha:
-    volume-size: 525
+    volume-size: 512
     instances: 2
     master: false
     instance: m6g.large.search
 
-  # 2 CPU per VM, 15GB RAM per VM, 1050GB disk space
+  # 4 vCPU, 16GB RAM, volume size range 10-1000GB
   large:
-    volume-size: 1050
+    volume-size: 1000
     instances: 1
     master: false
     instance: m6g.xlarge.search
 
-  # 3 dedicated VMs, 2 CPU per VM, 15GB RAM per VM, 1050GB disk space
+  # 2 nodes with 4 vCPU, 16GB RAM, volume size range 10-1000GB
   large-ha:
-    volume-size: 1050
+    volume-size: 1000
     instances: 2
     master: false
     instance: m6g.xlarge.search
 
-  # 4 CPU per VM, 31GB RAM per VM, 2100GB disk space
+  # 8 vCPU, 32GB RAM, volume size range 10-1500GB
   x-large:
-    volume-size: 2100
+    volume-size: 1500
     instances: 1
     master: false
     instance: m6g.2xlarge.search
 
-  # 3 dedicated VMs, 4 CPU per VM, 31GB RAM per VM, 2100GB disk space
+  # 2 nodes with 8 vCPU, 32GB RAM, volume size range 10-1500GB
   x-large-ha:
-    volume-size: 2100
+    volume-size: 1500
     instances: 2
     master: false
     instance: m6g.2xlarge.search
 
 # RDS/Postgres Instances
 rds-postgres:
-  # 5GB Storage, DB Instance Class: db.t3.micro
+  # 2v CPU, 1GB RAM, 5GB Storage
   tiny:
     volume-size: 5
     replicas: 0
```

Comment:
Not sure why a "replicas" integer is used here. In the Cloudformation template ouput it enables MultiAZ if the value is greater than 0, the actual number doesn't matter. It could probably be `multi-az: true|false` for more clarity.